.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -40,6 +41,14 @@ body:
     validations:
       required: true
 
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: A clear and concise description of what actually happened.
+    validations:
+      required: true
```

Comment:
Would you be able to outline the difference between this and the “Describe the bug” section (line 14)? It seems to me like only one of these two sections should be kept.

Reply:
Well, I simply thought that the "Actual Behavior" part was the supplement to the "Expected Behavior" part. Although now I realized that it may/may not be a duplicate of the first section & the title... (Also I got inspiration from other issue templates, and this was a common section.)

Should I remove it or not?

Reply:
Since describing the bug (first section) and describing the actual behavior (second section) are, as far as I can tell, the same thing, Iʼd suggest removing this section in order to avoid duplication.

Reply:
Sure, I'll remove it soon.